File: backend/src/main/java/vjda/resource/bean/ToDoBean.java
package vjda.resource.bean;

import java.io.Serializable;

import lombok.Data;

/** Data Transfer Object for request and response bodies of ToDoResource */
@Data public class ToDoBean implements Serializable {
	private Long id;
	private String title;
	private Category category;
	private String content;

	public enum Category { one, two, three }
}

File: backend/src/main/kotlin/backend/resource/bean/ToDoBean.kt
package backend.resource.bean

/** Data Transfer Object for request and response bodies of ToDoResource  */
data class ToDoBean (
    var id: Long? = null,
    var title: String? = null,
    var category: Category? = null,
    var content: String? = null
) {
    enum class Category {
        one, two, three
    }
}